queries/flights.py

```diff
@@ -112,21 +122,18 @@ def help_text(lemma: str) -> str:
     | "hver" "er" "brottfarartími"
     | "hver" "er" "brottfarartíminn" "fyrir"
 
-$score(+35) QFlightsWhenDep
-
 QFlightsWhen →
     "hvenær" | "klukkan" "hvað"
 
-$score(-35) QFlightsWhen
-
 QFlightsPlane →
     "flug" | "flugvél" | "flugvélar" | "vél" | "vélar" | "flugs"
 
-$score(+100) QFlightsPlane
-
 QFlightsPathDesc →
-    # Specify at least one endpoint
-    QFlightsPrepLoc QFlightsPrepLoc > QFlightsPrepLoc
+    # At least one endpoint of the flight (in any order), e.g.
+    # frá Keflavík til Reykjavíkur
+    # til Akureyrar
+    QFlightsPrepLoc QFlightsPrepLoc
+    > QFlightsPrepLoc
```

Comment:
Not totally sure that this is wise; more canonical is simply `QFlightsPrepLoc QFlightsPrepLoc?`, omitting the second production.

Reply:
I originally did that but kept running into the problem of queries such as "Hvenær fer næsta flug frá New York til Keflavík?" only using one QFlightsPrepLoc and interpreting 'New York til Keflavík' as a single location.
Do you have an idea how that could be fixed/circumvented?

Reply:
OK, I see your point. Strictly speaking we shouldn't be using the `Nl/fall` nonterminal in the main grammar; it should be one of the more restrictive sub-nonterminals underl `Nl` that doesn't allow a preposition after the noun phrase. But your original hack is an OK solution considering the complexity of the alternatives.